data/events.yaml

```diff
@@ -0,0 +1,31 @@
+x-events:
+  - name: cancel_install_failed_start
```

Comment:
@avishayt do we need event IDs or is the name enough?

Reply:
I removed the ID since I haven't noticed it is being used or persisted to the events table.
Unless it is required for a specific use, I'd postpone adding it to the future if/when it will be needed in order not to commit to it.

Reply:
Agree with @masayag.
However, we need to be conscious of the fact that users (including internal users like UI and operator) will rely on the name as an ID.  We'll need to maintain that contract.

Reply:
OK